### PR TITLE
Fix comment indentation 

### PR DESCRIPTION
**Pull request by shivam-tripathi**

### CODE REVIEW

**Comment by yanokwa** [comment]
Thanks for sending this in @shivam-tripathi! Can you commit your change to the checkstyle.xml as well? See https://github.com/opendatakit/briefcase/pull/111/files for an example.

File: src/org/opendatakit/aggregate/parser/BaseFormParserForJavaRosa.java [hunk 1]
```diff
@@ -124,10 +124,10 @@ private static final void initializeJavaRosa() {
   public static enum DifferenceResult {
     XFORMS_IDENTICAL, // instance and body are identical
     XFORMS_SHARE_INSTANCE, // instances (including binding) identical; body
-                           // differs
+    // differs
```
Comment (yanokwa) [inline src/org/opendatakit/aggregate/parser/BaseFormParserForJavaRosa.java]: Can we just put this on the same line?

File: src/org/opendatakit/aggregate/parser/BaseFormParserForJavaRosa.java [hunk 2]
```diff
@@ -124,10 +124,10 @@ private static final void initializeJavaRosa() {
...
     XFORMS_SHARE_SCHEMA, // instances differ, but share common database schema
     XFORMS_DIFFERENT, // instances differ significantly enough to affect
-                      // database schema
+    // database schema
```
Comment (yanokwa) [inline src/org/opendatakit/aggregate/parser/BaseFormParserForJavaRosa.java]: Can we just put this on the same line?

**Comment by shivam-tripathi** [comment]
Open for review

**Pull request merged**